frontend/src/components/common/SimpleTable.tsx

```diff
@@ -299,17 +299,20 @@ export default function SimpleTable(props: SimpleTableProps) {
           {filteredData.length > 0 ? (
             getPagedRows().map((row: any, i: number) => (
               <TableRow key={i}>
-                {columns.map((col, i) => (
-                  <TableCell key={`cell_${i}`}>
-                    {i === 0 && row.color && (
-                      <React.Fragment>
-                        <InlineIcon icon="mdi:square" color={row.color} height="15" width="15" />
-                        &nbsp;
-                      </React.Fragment>
-                    )}
-                    {'datum' in col ? row[col.datum] : col.getter(row)}
-                  </TableCell>
-                ))}
+                {columns.map((col, i) => {
+                  const { cellProps = {} } = col;
+                  return (
+                    <TableCell key={`cell_${i}`} {...cellProps}>
```

Comment:
ps. Using array index for the key is problematic. It causes problems when the data changes. (I know this problem was here before this change, just mentioning it)

Reply:
I am not aware of a better way to assign indexes in this case. The name for cells is not unique, so this sounds okay according with https://reactjs.org/docs/lists-and-keys.html .

Reply:
Yeah, should be fine.